cfgov/v1/util/util.py

```diff
@@ -126,6 +126,23 @@ def get_secondary_nav_items(request, current_page):
 
         nav_items.append(item)
 
+    # Add `/process/` segment to OAH journey page nav urls.
+    # TODO: Remove this when redirects for `/process/` urls
+    # are added after 2018 homebuying campaign.
+    journey_urls = (
+        '/owning-a-home/prepare',
+        '/owning-a-home/explore',
+        '/owning-a-home/compare',
+        '/owning-a-home/close',
+        '/owning-a-home/sources',
+    )
+    if current_page.relative_url(request.site).startswith(journey_urls):
+        for item in nav_items:
+            item['url'] = '/owning-a-home/process' + item['url'][14:]
```

Comment:
@virginiacc is there a way to avoid hardcoding `14` in? you could potentially split the string on `'owning-a-home/'` to be more explicit.

Reply:
Or, you could do a string.replace e.g. `item['url'].replace('owning-a-home', 'owning-a-home/process')`

Reply:
Good point, thanks! Updated in https://github.com/cfpb/cfgov-refresh/pull/3950/commits/e2e9adec7cac29b208ed58996f56043d4f3fb718